### PR TITLE
lang: [cs] Add Czech Translation

### PR DESCRIPTION
**Pull request by MichalSkoula**
I have added Czech translation for shield ~+ fixed Slovak translation~.

### CODE REVIEW

**Review by kenjis — CHANGES_REQUESTED** [review]
Please don't include [sk] changes in a [cs] PR.

> One thing at a time: A pull request should only contain one change. That does not mean only one commit, but one change - however many commits it took. 
https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#branching

**Comment by MichalSkoula** [comment]
Ok I have reverted Slovak translation with a new commit.

**Comment by datamweb** [comment]
Hi,
Thank you for contributing!
Please see https://github.com/codeigniter4/shield/pull/235#issuecomment-1153050805

**Comment by MichalSkoula** [comment]
ok done :)

File: src/Language/cs/Auth.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    'magicTokenNotFound' => 'Odkaz se nepodařilo ověřit.',
+    'magicLinkExpired'   => 'Litujeme, ale platnost odkazu vypršela.',
+    'checkYourEmail'     => 'Zkontrolujte si vaši e-mailovou schránku',
+    'magicLinkDetails'   => 'Právě jsme vám poslali e-mail s tajným odkazem pro přihlášení. Platí pouze po dobu {0} minut.',
+    'magicLinkDisabled'  => 'Použití tajného odkazu není v současné době povoleno.',
+    'successLogout'      => 'Odhlášení proběhlo úspěšně.',
+    'backToLogin'        => 'Zpět na přihlášení',
+
+    // Passwords
+    'errorPasswordLength'       => 'Heslo musí mít alespoň {0, číslo} znaků.',
```
Comment (datamweb) [inline src/Language/cs/Auth.php]:
```suggestion
    'errorPasswordLength'       => 'Heslo musí mít alespoň {0, number} znaků.',
```

File: src/Language/cs/Auth.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    'suggestPasswordLength'     => 'Hesla - dlouhá až 255 znaků - jsou bezpečnější a snadno zapamatovatelná.',
+    'errorPasswordCommon'       => 'Heslo nesmí být běžné heslo.',
+    'suggestPasswordCommon'     => 'Heslo bylo porovnáno s více než 65 tisíci běžně používanými hesly nebo hesly, která unikla prostřednictvím hackerů.',
+    'errorPasswordPersonal'     => 'Hesla nemohou obsahovat přepsané osobní údaje.',
+    'suggestPasswordPersonal'   => 'Jako hesla by se neměly používat varianty e-mailové adresy nebo uživatelského jména.',
+    'errorPasswordTooSimilar'   => 'Heslo je příliš podobné uživatelskému jménu.',
+    'suggestPasswordTooSimilar' => 'V heslu nepoužívejte části svého uživatelského jména.',
+    'errorPasswordPwned'        => 'Heslo {0} bylo odhaleno v důsledku úniku dat a bylo zaznamenáno {1, počet}krát z {2} prolomených hesel.',
```
Comment (datamweb) [inline src/Language/cs/Auth.php]:
```suggestion
    'errorPasswordPwned'        => 'Heslo {0} bylo odhaleno v důsledku úniku dat a bylo zaznamenáno {1, number} krát z {2} prolomených hesel.',
```

File: tests/Language/CzechTranslationTest.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Tests\Language;
+
+/**
+ * @internal
+ */
+final class CzechTranslationTest extends AbstractTranslationTestCase
+{
```
Comment (datamweb) [inline tests/Language/CzechTranslationTest.php:20]:
```suggestion
{
    protected array $excludedLocaleKeyTranslations = [
        'Auth.token',
    ];
```

**Review by datamweb — CHANGES_REQUESTED** [review]

File: src/Language/cs/Auth.php [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    'invalidJWT'     => 'Neplatný token.',
+    'expiredJWT'     => 'Platnost tokenu vypršela.',
+    'beforeValidJWT' => 'Token ještě není dostupný.',
+
+    'email'           => 'Emailová adresa',
+    'username'        => 'Uživatelské jméno',
+    'password'        => 'Heslo',
+    'passwordConfirm' => 'Heslo (znovu)',
+    'haveAccount'     => 'Máte už účet?',
+    'token'           => '(To be translated) Token',
```
Comment (datamweb) [inline src/Language/cs/Auth.php]:
```suggestion
    'token'           => 'Token',
```

**Review by datamweb — APPROVED** [review]
LGTM!
thank you!

**Review by kenjis — APPROVED** [review]

**Comment by datamweb** [comment]
@MichalSkoula See https://github.com/codeigniter4/shield/pull/672#issuecomment-1468658726

**Comment by MichalSkoula** [comment]
> @MichalSkoula See [#672 (comment)](https://github.com/codeigniter4/shield/pull/672#issuecomment-1468658726)

Should I create a new PR?

**Comment by datamweb** [comment]
How to sign past commits is explained [here](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#gpg-signing-old-commits), but feel free to do so if you're comfortable creating new PRs.

**Comment by MichalSkoula** [comment]
Creating a new PR is easier for me. I have done so - https://github.com/codeigniter4/shield/pull/1015

**Pull request closed without merging**